### ISSUE TITLE
[Adaptive Cards- Import- Expense Report]: Talkback does not announce the state of Dropdown controls after activation present under 'Expense' pane.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6975]
https://microsoft.visualstudio.com/OS/_workitems/edit/34056449
